requirements.txt

```diff
@@ -4,13 +4,15 @@ transformers==4.19
 tokenizers
 inflect
 progressbar
-einops
+einops==0.4.1
 unidecode
-scipy
-librosa
+scipy==0.10.1
```

Comment:
Version of scipy not 0.10.1, should be scipy==1.10.1

Reply:
I tested this with Colab and seems to work. Can you share a Colab link or raise a new PR that'd make it easier to test this?